src/en/credentials.md

```diff
@@ -93,16 +93,51 @@ notice if they change in future.
 
 ### Adding credentials from a YAML file.
 
-You can also specify a YAML format file for the credentials. This  
-file would be similar to:
+You can also specify a YAML format file for the credentials. This
+file would be similar to, but shorter than this extensive sample, which
+we will call mycreds.yaml:
 
 ```yaml
 credentials:
-  aws:
-    bob:
-      auth-type: access-key
-      access-key: AHJHKUWK7HIW
-      secret-key: 21f8cbb668263a1223755b5f15c48a
+      aws:
+        default-credential: peter
+        default-region: us-west-2
+        peter:
+          auth-type: access-key
+          access-key: key
+          secret-key: secret
+        paul:
+          auth-type: access-key
+          access-key: key
+          secret-key: secret
+      homemaas:
+        peter:
+          auth-type: oauth1
+          maas-oauth: mass-oauth-key
+      homestack:
+        default-region: region-a
+        peter:
+          auth-type: userpass
+          password: secret
+          tenant-name: tenant
+          username: user
+      google:
+        peter:
+          auth-type: jsonfile
+          file: path-to-json-file
+      azure:
+        peter:
+          auth-type: userpass
+          application-id: blah
+          subscription-id: blah
+          application-password: blah
+      joyent:
+        peter:
+          auth-type: userpass
+          sdc-user: blah
+          sdc-key-id: blah
+          private-key: blah (or private-key-path)
+          algorithm: blah
 ```
```

Comment:
I'm not sure about the values here. I appreciate that it is easy to see that you have to enter something. I quite like the 'dummy' value to look like a real one - it makes it easier to see what bits go where etc.